src/components/Filters/PollIntervalFilter.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+
+import { config } from '../../config';
+import { ToolbarDropdown } from '../ToolbarDropdown/ToolbarDropdown';
+import { PollInterval } from '../../types/GraphFilter';
+
+type PollIntervalFilterProps = {
+  id: string;
+  disabled: boolean;
+  onPollIntervalChanged: (interval: number) => void;
+  selected?: PollInterval;
+};
+
+const PollIntervals = config().toolbar.pollInterval;
+const DefaultPollInterval = config().toolbar.defaultPollInterval;
+
+const PollIntervalFilter: React.SFC<PollIntervalFilterProps> = props => {
+  const selectedLabel = PollIntervals[props.selected!.value]
+    ? PollIntervals[props.selected!.value]
+    : `${props.selected!.value} ms`;
+  return (
+    <ToolbarDropdown
+      id={props.id}
+      disabled={props.disabled}
+      handleSelect={props.onPollIntervalChanged}
+      nameDropdown="Poll Interval"
+      initialValue={props.selected!.value}
+      initialLabel={selectedLabel}
+      options={PollIntervals}
+    />
+  );
+};
+
+PollIntervalFilter.defaultProps = {
+  selected: { value: DefaultPollInterval }
```

Comment:
I don't know that the extra object inside `selected` property is necessary, referring to the `value`. Unless there are multiple properties (like: `unitOfMeasure` or others) it just seems like it's not adding much.  The `.value` portion seems redundant. `defaultProps.selected` is all we need (which would have the type `DefaultPollInterval`)
It's a minor nit, and maybe I'm just not thinking of something.
I think there are no changes needed here as this has been done in other work -- you were just following one precedent -- although there is a mixture.
We can address this separately in a Kiali-dev mailing.